integration-cli/docker_api_exec_test.go

```diff
@@ -108,6 +99,30 @@ func (s *DockerSuite) TestExecAPIStartBackwardsCompatible(c *check.C) {
 	c.Assert(resp.StatusCode, checker.Equals, http.StatusOK, comment)
 }
 
+// #19362
```

Comment:
is it even necessary? git blame can give you the commit and you can trace that back to the PR and issue in github.